microsoft/DiagManager
# Multiple Issues fixed in this PR (see issue list)

Fixes # .
https://github.com/microsoft/DiagManager/issues/169
https://github.com/microsoft/DiagManager/issues/73
https://github.com/microsoft/DiagManager/issues/104
https://github.com/microsoft/DiagManager/issues/175
https://github.com/microsoft/DiagManager/issues/150
https://github.com/microsoft/DiagManager/issues/78
https://github.com/microsoft/DiagManager/issues/74
https://github.com/microsoft/DiagManager/issues/110
https://github.com/microsoft/DiagManager/issues/85
https://github.com/microsoft/DiagManager/issues/44 


How to test this code:
 - Run varios pssdiag collections and confirm the respective data is collected for each issue.

Has been tested on (remove any that don't apply):
 - SQL Server 2012
 - SQL Server 2014
 - SQL Server 2016
 - SQL Server 2017
 - SQL Server 2019
